string type      # either "name", "location" or "object"
uint32 n_tries
float32 time_out
---
string answer
